test/qweb3.js

```diff
@@ -512,4 +512,19 @@ describe('Qweb3', () => {
       });
     });
   });
+
+  describe('backupWallet()', () => {
+    it('backup the wallet', async () => {
+      const res = await qweb3.backupWallet('backup.dat');
```

Comment:
use `path` module to save `backup.dat` in `qweb3/test/data/backup.dat`. use relative path to get there.